opendatahub-io/odh-dashboard
# Action to create quay tag from git tag

Resolves: #588 

## Description
Adds a github action that uses skopeo to copy the quay.io image of main-<hash>
 (where hash is the latest hash on github at the time of tag push) to quay io image tag of <git-tag>

NOTE: This action expects that the `QUAY_ROBOT_TOKEN` is present on this repository, containing the token for the robot account by the name of `opendatahub+odh-dashboard` for quay.io, this can be changed before merging ofcourse.

## How Has This Been Tested?
Verified on my fork of odh-dashboard:
1. Ensure corresponding main-<latest git abbrev hash> tag is present on quay.io
2. Publish a tag to github
3. Action should correctly run and create a copy of image with corresponding tag in quay.io

## Merge criteria:
